arviz/plots/densityplot.py

```diff
@@ -28,6 +28,7 @@ def plot_density(
     bw=4.5,
     figsize=None,
     textsize=None,
+    group="posterior",
```

Comment:
if you've got a moment, could you move this up to be the 2rd argument (and make sure it matches that order in the documentation)? It feels like this will be a helpful general change everywhere, and feels like it should be the second argument.